src/components/UpdateStudentName.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import EasyEdit from "react-easy-edit";
+import axios from "axios";
+import { withSnackbar } from "notistack";
+
+
+const baseUrl = process.env.API_URL;
+
+export class UpdateStudentName extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  handleUpdate = (name) => {
+    const { rowMetatable, change } = this.props;
+    const studentId = rowMetatable.rowData[0];
+    axios
+      .put(`${baseUrl}students/${studentId}`, { name })
+      .then(() => {
+        console.log("Success");
+        this.props.enqueueSnackbar("Name updated successfully!", {
+          variant: "success",
+        });
+        change(name);
+      })
+      .catch(() => {
+        console.log("Failed");
+        this.props.enqueueSnackbar("Couldn't update name !", {
+          variant: "error",
+        });
+      });
+  };
+
+  render = () => {
+    return (
+      <div>
+        <EasyEdit
```

Comment:
@komalng I believe we already use a library for editing the emails. It's best to use the same and avoid adding extra libraries for doing the same task..

Reply:
Yes, I am using the same library.